sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -347,7 +347,8 @@ public BlobInputStream openInputStream(BlobInputStreamOptions options, Context c
             new com.azure.storage.common.ParallelTransferOptions().setBlockSizeLong((long) chunkSize);
         BiFunction<BlobRange, BlobRequestConditions, Mono<BlobDownloadAsyncResponse>> downloadFunc =
             (chunkRange, conditions) -> client.downloadStreamWithResponse(chunkRange, null, conditions, false, contextFinal);
-        return ChunkedDownloadUtils.downloadFirstChunk(range, parallelTransferOptions, requestConditions, downloadFunc, true)
```

Comment:
It might be worth double checking if `openInputStream` is impacted by the same issue as `downloadToFile`. I have not checked it, but would be good to double check since it is relying on the `ChunkedDownloadUtils` class and also see if we have any tests for this method in our tests.

Reply:
openInputStream is not impacted by the same issue, and works without the context adjustment passed in. I have added a test for it.